crates/runtime-configs/Cargo.toml

```diff
@@ -15,11 +15,13 @@ slight-runtime = { path = "../runtime" }
 argon2rs = "0.2.5"
 rand = "0.8.5"
 crossbeam-channel = "0.5.5"
-uuid = "1"
+uuid = { version = "1.1.2", features = ["v4"] }
 spiderlightning = { path = "../.." }
 slight-events-api = { path = "../events-api" }
 toml = "0.5.9"
 short-crypt = "1"
+azure-app-configuration = { git = "https://github.com/danbugs/azure-app-configuration.git", rev = "3c6c0886840847e98e66e9189f540d1db97fe316" }
```

Comment:
Nice! Will you contribute this to the upstream `azure-sdk-for-rust`?

Reply:
This isn't part of `azure-sdk-for-rust` ~ might contribute it to its' original repo, though